core/scss/utilities/mixins/typography/_intro-text.scss

```diff
@@ -1,16 +1,17 @@
 @charset "UTF-8";
 
 //
-// @font-lead
+// @intro-text
 //
 // For leading a page of text, summarizing a main point, and/or highlighting a
 // paragraph or quote.
 //
-// Style guide: Mixins.Typography.font-lead
+// Style guide: Mixins.Typography.intro-text
 //
-@mixin font-lead {
+@mixin intro-text {
   @include modular-typography(2);
+
   font-weight: $font-light;
```

Comment:
```suggestion
  font-weight: $font-normal;
```

Reply:
It has a normal font weight in Figma, but with font-smoothing it would look lighter.

Reply:
Called `$font-regular` now to match Figma and Google. I updated this. :)